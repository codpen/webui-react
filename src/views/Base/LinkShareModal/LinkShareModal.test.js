import React from "react";

// const setUp = (intialState = {}, props = {}) => {
//     const store = testStore(intialState);
//     const component = shallow(<LinkShareModal {...props} store={store}/>);
//     return component.childAt(0).dive();
// }
//
// describe('Link Share Modal', function () {
//
//
//     describe('renders', function () {
//         let wrapper;
//         beforeEach(() => {
//             const props = {
//                 isVisible: true,
//                 closeModal: jest.fn(),
//                 linkUrl: ""
//             };
//             const initialState = {};
//             wrapper = setUp(initialState, props)
//         });
//
//         it('should render without crashing', function () {
//             expect(wrapper).toHaveLength(1)
//         });
//
//         it('should match snapshot', function () {
//             expect(toJson(wrapper)).toMatchSnapshot()
//         });
//     });
//

// });

it('should 1 === 1', function () {
    expect(1).toBe(1)
});