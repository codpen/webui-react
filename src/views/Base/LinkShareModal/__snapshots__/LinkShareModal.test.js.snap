// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link Share Modal renders should match snapshot 1`] = `
<Portal
  node={
    <div
      style="position: relative; z-index: 1050;"
      tabindex="-1"
    />
  }
>
  <div
    className=""
  >
    <Fade
      appear={true}
      baseClass="fade"
      baseClassActive="show"
      className="modal"
      enter={true}
      exit={true}
      in={true}
      mountOnEnter={false}
      onClick={[Function]}
      onEnter={[Function]}
      onEntered={[Function]}
      onEntering={[Function]}
      onExit={[Function]}
      onExited={[Function]}
      onExiting={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      role="dialog"
      style={
        Object {
          "display": "block",
        }
      }
      tabIndex="-1"
      tag="div"
      timeout={300}
      unmountOnExit={false}
    >
      <div
        className="modal-dialog"
        role="document"
      >
        <div
          className="modal-content"
        >
          <ModalHeader
            charCode={215}
            closeAriaLabel="Close"
            data-test="modalHeader"
            tag="h5"
            toggle={[Function]}
            wrapTag="div"
          >
            Public link generated.
          </ModalHeader>
          <ModalBody
            data-test="modalBody"
            tag="div"
          >
            <Input
              onFocus={[Function]}
              readOnly={true}
              type="text"
              value=""
            />
            <div>
              <Button
                color="link"
                onClick={[Function]}
                tag="button"
              >
                <i
                  className="fa fa-clipboard fa-2x"
                />
              </Button>
            </div>
          </ModalBody>
          <ModalFooter
            data-test="modalFooter"
            tag="div"
          >
            <Button
              color="primary"
              onClick={[Function]}
              tag="button"
            >
              Done
            </Button>
             
          </ModalFooter>
        </div>
      </div>
    </Fade>
    <Fade
      appear={true}
      baseClass="fade"
      baseClassActive="show"
      className="modal-backdrop"
      enter={true}
      exit={true}
      in={true}
      mountOnEnter={true}
      onEnter={[Function]}
      onEntered={[Function]}
      onEntering={[Function]}
      onExit={[Function]}
      onExited={[Function]}
      onExiting={[Function]}
      tag="div"
      timeout={150}
      unmountOnExit={false}
    />
  </div>
</Portal>
`;
